themes/rjorgenson/rjorgenson.theme.sh

```diff
@@ -14,88 +14,82 @@ SCM_THEME_PROMPT_CLEAN=" ${bold_green}✓${normal}"
 SCM_GIT_CHAR="${STRING_COLOR}±${normal}"
 SCM_SVN_CHAR="${bold_cyan}⑆${normal}"
 SCM_HG_CHAR="${bold_red}☿${normal}"
+PROMP_CHAR="${BRACKET}➞ ${normal}"
 
 #Mysql Prompt
 export MYSQL_PS1="(\u@\h) [\d]> "
 
+TITLEBAR=""
 case $TERM in
-        xterm*)
-        TITLEBAR="\[\033]0;\w\007\]"
-        ;;
-        *)
-        TITLEBAR=""
-        ;;
+        xterm*) TITLEBAR="\[\033]0;\w\007\]" ;;
 esac
 
-PS3=">> "
+if [[ $OSTYPE =~ "darwin" ]]; then
+    echo "${BRACKET}➞  ${normal}"
```

Comment:
This seems to output the above string on the startup of the shell session. Is this intended? The original code was used to put the string in the prompt.

Ah, OK. I think you have forgot to replace `echo` with `PROMP_CHAR=`. We can move this code just below the assignment of the default value to `PROMP_CHAR`.

Reply:
You are correct. I don't have/use a Mac so I didn't catch the issue.